src/main/groovy/com/github/lburgazzoli/gradle/plugin/karaf/features/KarafFeaturesTask.groovy

```diff
@@ -80,10 +80,18 @@ class KarafFeaturesTask extends DefaultTask implements KarafTaskTrait  {
 
                     feature.configFiles.each {
                         if (it.filename && it.uri) {
-                            builder.configfile(
-                                [ finalname: it.filename ],
-                                it.uri
-                            )
+                            if (it.override) {
+                                builder.configfile(
```

Comment:
Maybe it would be better to have FeatureDefinition.ConfigFile.override as Boolean so this does not require the if/else and if one explicit sets it, it will be also explicit written to the feature file